src/rastervision/samples/workflow-configs/segmentation/deeplab-test.json

```diff
@@ -0,0 +1,110 @@
+{
+    "train_scenes": [
+        {
+            "id": "2-10",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_10_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_10_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "raster_class_map": [
+                        {
+                            "source_class": "#ffff00",
+                            "raster_vision_class": 255
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "test_scenes": [
+        {
+            "id": "2-11",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_11_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_11_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "raster_class_map": [
```

Comment:
The `raster_class_map` is easier to understand. But, I still don't know why there is only an item for car, and not for building. Also, class_ids are expected to go from 1 to N for object detection and classification or things will break, and that should be fixed at some point. So I'm glad your implementation doesn't depend on it (since you've chosen 127 and 255 as ids), but that could be confusing, so maybe they should be changed to 1 and 2?

Reply:
Isn't this sample only detecting cars? There are several classes inside the labels (building, low veg, trees, clutter, impervious surface) - bit confused why we would include buildings but not all of these, and I  can see including just the target class, in this case, cars

Reply:
The `class_items` contains items for cars and buildings, but the `raster_class_map` only has an item for car. The mismatch is what's confusing me. I think it makes sense to use a (consistent) subset though.